ci/cloudbuild/cloudbuild.yaml

```diff
@@ -48,6 +49,8 @@ availableSecrets:
   - versionName: projects/${PROJECT_ID}/secrets/CODECOV_TOKEN/versions/latest
     env: 'CODECOV_TOKEN'
 
+logsBucket: 'gs://${PROJECT_ID}_${_LOGS_BUCKET_SUFFIX}/logs/google-cloud-cpp/${_CACHE_TYPE}/${COMMIT_SHA}/${_DISTRO}-${_BUILD_NAME}'
```

Comment:
nit: the use of `_CACHE_TYPE` sounds odd...

Reply:
Yep. Re-using it here suggests that it's a bad name. It results to a PR number for PR builds, `main` for CI builds, and `manual-user@domain` for manually triggered builds. I'll look into renaming that to something clearer in a separate PR.